include/aie/Conversion/Passes.td

```diff
@@ -14,7 +14,7 @@ include "mlir/Pass/PassBase.td"
 //===----------------------------------------------------------------------===//
 // AIEVecToLLVM
 //===----------------------------------------------------------------------===//
-def ConvertAIEVecToLLVM : Pass<"convert-aievec-to-llvm", "ModuleOp"> {
+def ConvertAIEVecToLLVM : Pass<"lower-aievec", "ModuleOp"> {
```

Comment:
Why make this change?  I think 'convert....' is the usual naming convention for such passes?

Reply:
@brandon-t-nguyen: I'll update the diagram; please use the standard name.